hack/verify-pylint.sh

```diff
@@ -17,8 +17,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export PYLINTHOME=$(mktemp -d)
-pylint="$(dirname $0)/pylint_bin"
+export PYLINTHOME
+PYLINTHOME=$(mktemp -d)
```

Comment:
huh, why is it preferable to do this?

Reply:
The general warning is that `export A=$(foo)` succeeds even if foo fails.

Reply:
Thanks. I should really setup shellcheck locally, I bet a lot of other bash in the project could use these fixes.
Filing an issue to figure out how we can integrate it.

Reply:
Yeah this is a really common issue -- you have two return codes:

```sh
export MYVAR="$( some_expression )"
^-----A-----^ ^---------B---------^
```

So if `B` fails but `A` succeeds the line does not error as you would expect (return code of `A` shadows that of `B`). This happens on all scoping statements (`export`, `local`, `readonly`)